[CI] Rest text `p0=snapshot/10_basic/Create a source only snapshot and then restore it` failure

Received this build failure during the push of an unrelated docs change (a311018fbcac72f479be29bef32bfb312193dbce)

Failure: https://gradle-enterprise.elastic.co/s/v5h5hntmwkiqy/tests/fcijb24gqtmnc-cqpxjyuvwnxpc

Based on the results, it looks like two snapshots were restored instead of one as expected.

```
Caused by: java.lang.AssertionError: 
snapshot.shards.successful didn't match expected value:
    snapshot.shards.successful: expected Integer [1] but was Integer [2]
```

```[2019-12-20T22:16:11,776][INFO ][o.e.x.t.r.XPackRestIT    ] [test] Stash dump on test failure [{
  "stash" : {
    "body" : {
      "snapshot" : {
        "snapshot" : "test_snapshot",
        "uuid" : "1pr60of1RIWn1w2MAjRmRw",
        "version_id" : 8000099,
        "version" : "8.0.0",
        "indices" : [
          "test_index",
          "ilm-history-1-000001"
        ],
        "include_global_state" : true,
        "state" : "SUCCESS",
        "start_time" : "2019-12-20T19:16:11.335Z",
        "start_time_in_millis" : 1576869371335,
        "end_time" : "2019-12-20T19:16:11.535Z",
        "end_time_in_millis" : 1576869371535,
        "duration_in_millis" : 200,
        "failures" : [ ],
        "shards" : {
          "total" : 2,
          "failed" : 0,
          "successful" : 2
        }
      }
    }
  }
}]
```

--- a/x-pack/plugin/src/test/resources/rest-api-spec/test/snapshot/10_basic.yml
+++ b/x-pack/plugin/src/test/resources/rest-api-spec/test/snapshot/10_basic.yml
@@ -39,6 +39,8 @@ setup:
         repository: test_repo_restore_1
         snapshot: test_snapshot
         wait_for_completion: true
+        body: |
+          { "indices": "test_index" }
 
   - match: { snapshot.snapshot: test_snapshot }
   - match: { snapshot.state : SUCCESS }

